hal/src/main/java/edu/wpi/first/wpilibj/hal/FRCNetComm.java

```diff
@@ -12,160 +12,168 @@
 /**
  * JNI wrapper for library <b>FRC_NetworkCommunication</b><br>.
  */
-@SuppressWarnings("MethodName")
+@SuppressWarnings({"MethodName", "LineLength"})
 public class FRCNetComm extends JNIWrapper {
   /**
    * Module type from LoadOut.h
    */
   @SuppressWarnings("TypeName")
-  public interface tModuleType {
-    int kModuleType_Unknown = 0x00;
-    int kModuleType_Analog = 0x01;
-    int kModuleType_Digital = 0x02;
-    int kModuleType_Solenoid = 0x03;
+  public static final class tModuleType {
+    private tModuleType() { }
```

Comment:
Checkstyle should get mad at the braces being on the same line

Reply:
That's odd. I've had checkstyle complain at me before about putting open and close braces on the same line for empty functions. I wonder why it didn't complain here.

Reply:
clang-format would remove the space between the braces, by the way.

Reply:
Yeah I don't know why it didn't complain about either issue. And the reason for the space is I couldn't figure out how to escape the `{}` in python

Reply:
You don't need to escape `{}`. The script works just fine without the space.